cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/export/CgmesExportUtil.java

```diff
@@ -42,6 +43,8 @@ private CgmesExportUtil() {
     private static final DecimalFormat DOUBLE_FORMAT = new DecimalFormat("0.##############", DOUBLE_FORMAT_SYMBOLS);
     private static final DecimalFormat SCIENFIFIC_FORMAT = new DecimalFormat("0.####E0", DOUBLE_FORMAT_SYMBOLS);
 
+    private static final Pattern ID_PATTERN = Pattern.compile("_[a-f0-9]{8}-[a-f0-9]{4}-4[a-f0-9]{3}-[89aAbB][a-f0-9]{3}-[a-f0-9]{12}");
```

Comment:
You're right, this should be changed, I don't remember why I put that...

Reply:
I've added the case insensitivity, allowed all versions of UUID and added two patterns for valid IDs that are found in published ENTSO-e boundary files (UUID without dashes and UUIDS with final group of 7 digits instead of 12).